docs/update-from-client-side-helpers-to-server-side.md

```diff
@@ -0,0 +1,229 @@
+# Upgrading your Client-side integration to a Server-side integration
+
+We recommend using a combination of client-side and server-side code to integrate the PayPal Buttons on your e-commerce website. If you currently have client-side only PayPal Buttons integration, this page will describe how to update your integration to the recommended approach.
+<br/><br/>
+> 
+> **Know before you code**
+>  
+> [How to Setup a Developer Account](https://www.youtube.com/watch?v=O_9G722SpXQ&t=72s)
+> <br/><br/>
+
+# What do I need to change?
+
+If you are reading this page, we expect that you already have a client-side PayPal Button integration in place.  That said, several steps are required to get you up and running with the recommended approach. 
+
+Specifically:
+
+1. Setup a server-side integration with the [PayPal REST API's](https://developer.paypal.com/api/rest/).
+
+1. `createOrder` JavaScript callback must be changed to use your server-side to create and return an Order ID using the PayPal backend API's.
+
+1. `onApprove` JavaScript callback must be changed to use your server-side to complete transactions using the PayPal backend API's.
+
+<br/><br/>
+
+# 1. Setup a server-side integration with the PayPal REST API's
+
+PayPal REST API's follow a common integration approach used by other RESTful API's out there in the world.  For PayPal, you will minimally need to do the following:
+
+1. Setup a PayPal developer account if you have not already.
+
+1. Create a PayPal REST application and obtain a client + secret
+
+1. Use the client/secret combination to obtain an oAuth access token
+
+1. Use the oAuth access token to directly make REST calls to the PayPal Order API's -- which you will need for this document.
```

Comment:
```suggestion
1. Use the OAuth access token to directly make REST calls to the PayPal Order API's -- which you will need for this document.
```

I think the O is capitalized at least haha someone confirm with me that I'm not crazy